WordPress/WordPressIntents/en.lproj/Sites.strings

```diff
@@ -0,0 +1,12 @@
+"BOl9KQ" = "There are ${count} options matching ‘${site}’.";
+
+"ILcGmf" = "Site";
+
+"cyajMn" = "Site";
+
+"gpCwrM" = "Select Site";
+
+"s4dJhx" = "Just to confirm, you wanted ‘${site}’?";
```

Comment:
I didn't know this was possible to inject variables like that via `${site}`. I'm guessing this syntax is Intent-specific and the iOS runtime will automatically pick up the Intent's parameter with that name in those cases?

Reply:
I didn't either – this was auto-generated by Xcode though, and like you say, appears to be intent-specific.